src/mappings/geyser.ts

```diff
@@ -217,6 +217,21 @@ export function handleRewardsFunded(event: RewardsFunded): void {
     geyser.end = end;
   }
 
+  // create funding
+  let fundingId = geyser.id + '_' + event.block.timestamp.toString();
+  let funding = new Funding(fundingId);
+  funding.geyser = geyser.id;
+  funding.token = rewardToken.id;
+  funding.createdTimestamp = event.block.timestamp;
+  funding.start = event.params.start;
+  funding.end = event.params.start.plus(event.params.duration);
+  let formattedAmount = integerToDecimal(event.params.amount, rewardToken.decimals);
+  let shares = formattedAmount.times(geyser.sharesPerToken);
```

Comment:
this `sharesPerToken` value is actually for the staking side, we need rewards here. This is related to the first comment

Reply:
Great call, I wasn't getting this at first but I see now. Latest commit should address the staking vs reward sharesPerToken